ui/src/components/ui/PageHeader.scss

```
@@ -32,5 +32,8 @@
     display: flex;
     gap: $pad-small;
     align-items: center;
+    // Keep the actions/filter bar right-aligned whether or not the header carries a title. With space-between this is redundant when a
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)